docs/60_protocol-guides/30_network_peer_protocol.md

```diff
@@ -533,6 +542,8 @@ Therefore, the node’s LIB block is updated first, followed by the node’s hea
 
 Case 1 above, where the node’s LIB block needs to catch up with the peer’s LIB block, is depicted in the below diagram, before and after the sync (Note: inapplicable branches have been removed for clarity):
```

Comment:
Does the "Case 1" refer to step 1 in 3.3 Catch-Up Mode? If yes, maybe we can call them as "cases" or "steps." I had to search above where is Case 1. Upon reading I understood that step 1 is referred to as "Case 1."

Reply:
Good catch. Should be "Steps". I will update.